src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private static final double HALF = 0.5;
+    private double height;
+    private double side;
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Right Triangle, area: "
+                + getArea() + "sq. units, side "
+                + side + " units, height " + height
+                + " units, color: " + getColor().name());
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. It seems that `draw()` and `getArea()` methods can be separated into different interfaces as they serve different purposes.